sdk/storage/inc/blobs/blob_responses.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include "protocol/blob_rest_client.hpp"
+
+#include <limits>
+#include <string>
+#include <utility>
+
+namespace Azure { namespace Storage { namespace Blobs {
+
+  struct BlobDownloadInfo
```

Comment:
`DeleteContainerInfo`
`DeleteBlobInfo`
`UndeleteBlobInfo`
Those are all `<Verb><Object>Info`
But `BlobDownloadInfo` is `<Object><Verb>Info`.

Reply:
align with .Net SDK

Reply:
Then I don't think .NET SDK is doing the correct thing here. @JeffreyRichter what do you think? Is it going to be challenged by the review board in the future?

Reply:
I think it's fine.